src/main/java/org/amahi/anywhere/fragment/SettingsFragment.java

```diff
@@ -188,7 +189,12 @@ private void setUpApplicationVersion() {
 
 	private void setUpApplicationFeedback() {
 		Intent intent = Intents.Builder.with(getActivity()).buildFeedbackIntent();
-		startActivity(intent);
+		if (intent.resolveActivity(getActivity().getPackageManager()) != null) {
+			startActivity(intent);
+		}
+		else {
+			Toast.makeText(getActivity(), "NO Application Found", Toast.LENGTH_SHORT).show();
```

Comment:
Please declare the message in `Strings.xml` and use the xml resource here. This will help in translation process whenever it is done!